### DISCUSSION TITLE
I can't fork a repository

### DISCUSSION THREAD

**Discussion by insefo** [community/community#116724] [New to GitHub] [answered]
I can't fork any repository. I click fork, leave all the defaults (i'm the owner), and click create. It load a second, and returns me back to the main repository, without creating a fork for me.

I tried it with `gh repo fork` and get this error message: "failed to fork: HTTP 403: You cannot fork this repository at this time".

I have done this for multiple repositories, without any luck.

Anyone have any thoughts?

**Comment by insefo** [comment, marked as answer]
Solved. My account was flagged. You can check if you account is flagged, and request an appeal at the following link. https://support.github.com/request

**Reply by melroy89** [reply]
Sometimes you also can get blocked, causing you to not able to fork a repo anymore ;(

**Comment by akrigator** [comment]
@insefo, how to check is account is flagged and flagged means?

**Comment by krishnenduash** [comment]
all time show this error
